gems/dogstatsd-ruby/5.5.0/dogstatsd-ruby.rbs

```diff
@@ -0,0 +1,33 @@
+# Write the type definition here!
+
+module Datadog
+  class Statsd
+    # Create a new instance
+    #
+    # @param [String] host your statsd host
+    # @param [Integer] port your statsd port
+    # @option [String] namespace set a namespace to be prepended to every metric name
+    # @option [Array<String>|Hash] tags tags to be added to every metric
+    # @option [Logger] logger for debugging
+    # @option [Integer] buffer_max_payload_size max bytes to buffer
+    # @option [Integer] buffer_max_pool_size max messages to buffer
+    # @option [Integer] sender_queue_size size of the sender queue in number of buffers (multi-thread only)
+    # @option [Numeric] buffer_flush_interval interval in second to flush buffer
+    # @option [String] socket_path unix socket path
+    # @option [Float] default sample rate if not overridden
+    # @option [Boolean] single_thread flushes the metrics on the main thread instead of in a companion thread
+    def self.new: (
```

Comment:
It should be `initialize` instead of `new`. In Ruby, `new` class method is automatically defined if `initialize` instance method is defined. RBS defines it same as Ruby.

```suggestion
    def initialize: (
```